configstack/running_module.go

```diff
@@ -147,14 +147,15 @@ func removeFlagExcluded(modules map[string]*runningModule) map[string]*runningMo
 // Run the given map of module path to runningModule. To "run" a module, execute the RunTerragrunt command in its
 // TerragruntOptions object. The modules will be executed in an order determined by their inter-dependencies, using
 // as much concurrency as possible.
-func runModules(modules map[string]*runningModule) error {
+func runModules(modules map[string]*runningModule, concurrency int) error {
 	var waitGroup sync.WaitGroup
+	var s = make(chan struct{}, concurrency)
```

Comment:
Please use a more descriptive name than `s`. I can't guess what that stands for?

Reply:
fixed